doc/source/conf.py

```diff
@@ -18,29 +20,17 @@
     __version__ = file.read().splitlines()[0]
 release = version = __version__
 
-
-def get_version_match(semver):
-    """Evaluate the version match for the multi-documentation."""
-    if semver.endswith("dev0"):
-        return "dev"
-    major, minor, _ = semver.split(".")
-    return ".".join([major, minor])
-
-
 # Use the default pyansys logo
 html_logo = ansys_logo_black
 html_theme = "ansys_sphinx_theme"
 html_favicon = ansys_favicon
 
 # Specify the location of your GitHub repo
-version_mapper = (
-    "https://raw.githubusercontent.com/pyansys/actions/gh-pages/release/versions.json"
-)
 html_theme_options = {
     "github_url": "https://github.com/pyansys/actions",
-    "show_prev_next": False,
+    "show_prev_next": True,
```

Comment:
Interesting... I thought we weren't using this feature. In any case, if you want to set it to ``True`` there is no need to do that. It is set by default to ``True``. See https://pydata-sphinx-theme.readthedocs.io/en/v0.9.0/user_guide/configuring.html?highlight=show_prev_next#hiding-the-previous-and-next-buttons

Reply:
Oh! I see... Then I guess we can get rid of this line then.